samples/hssi/hssi_100g_cmd.h

```diff
@@ -162,20 +397,41 @@ class hssi_100g_cmd : public hssi_cmd
       else
         enable_eth_loopback(eth_ifc, false);
     }
+    ctrl_config config_data;
 
-    hafu->mbox_write(CSR_CTRL1, STOP_BITS);
+    uint64_t feature_rev;
```

Comment:
struct DFH {
    union {
        uint64_t csr;
        struct {
            uint64_t id : 12;
            uint64_t major_rev : 4;
            uint64_t next : 24;
            uint64_t eol : 1;
            uint64_t reserved41 : 7;
            uint64_t minor_rev : 4;
            uint64_t version : 8;
            uint64_t type : 4;
        };
    };
};
        
      struct DFH dfh;
        dfh.csr = hafu->read64(ETH_AFU_DFH);
        std::cout << "dfh.version "<< dfh.major_rev << std::endl;
        if ((dfh.major_rev != 2) || (!continuous_  == "on"|| (contmontitor_ == "off" )))
        {
            std::cout << "HSSI doesn't support continuous mode" << std::endl;
            return test_afu::error;
        }

Reply:
I have slightly modified the conditions to print "hssi doesnt support continous mode and exit" . i am currently testing these changes with rev 1 and rev2 bitfiles.  mostly we will push the revised changes by tomorrow.